### PR TITLE
[Translation] [Indonesian]: /id/advanced/proxy.md

### PR DESCRIPTION
**Pull request by CikiMomogi**
Ready to review.

### CODE REVIEW

File: site/docs/id/advanced/proxy.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+prev: ./transformers.md
+next: ./deployment.md
+---
+
+# Dukungan Proxy
+
+Di grammY, kamu bisa mengatur bagaimana network request dilakukan.
+Termasuk menambahkan payload khusus ke dalam setiap request yang akan digunakan untuk pemasangan proxy agent.
```
Comment (fadzikri) [inline site/docs/id/advanced/proxy.md:9]:
```suggestion
Termasuk menambahkan payload khusus ke setiap request yang akan digunakan untuk pemasangan proxy agent.
```
Aku nge-baca-in, `ke dalam setiap` rasa aneh gitu.

File: site/docs/id/advanced/proxy.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      // @ts-ignore
+      client,
+    },
+  },
+});
+```
+
+> Ingat! Gunakan flag `--unstable` supaya script di atas bisa berjalan.
+
+Berikut cara menggunakan proxy menggunakan package `socks5-https-client` ([npm](https://www.npmjs.com/package/socks-proxy-agent)) di Node.js:
```
Comment (fadzikri) [inline site/docs/id/advanced/proxy.md:32]:
```suggestion
Berikut cara menggunakan proxy dengan package `socks5-https-client` ([npm](https://www.npmjs.com/package/socks-proxy-agent)) di Node.js:
```
`menggunakan`-nya 2 kali diulang.

**Review by fadzikri — CHANGES_REQUESTED** [review]
Hey, check this out.

**Review by fadzikri — APPROVED** [review]
Ok.

File: site/docs/id/advanced/proxy.md (hunk 1 above)
Comment (CikiMomogi) [reply site/docs/id/advanced/proxy.md:9]: Ok!

File: site/docs/id/advanced/proxy.md (hunk 2 above)
Comment (CikiMomogi) [reply site/docs/id/advanced/proxy.md:32]: Oh my. How come I hadn’t noticed that?!

**Pull request merged**